lib/fab_circular_menu.dart

```diff
@@ -1,5 +1,7 @@
 library fab_circular_menu;
 
+import 'dart:ffi';
+
```

Comment:
This import is unused and must be removed

Reply:
Ugh my vscode and plugins automatically imports things and I don’t always notice